Marlin/src/sd/cardreader.cpp

```diff
@@ -483,7 +483,6 @@ void CardReader::manage_media() {
   if (ui.detected()) {
 
     uint8_t old_stat = prev_stat;
-    prev_stat = stat;                 // Change now to prevent re-entry
```

Comment:
This must be done before any call to `safe_delay` or other functions that could call `idle()` and end up back here.

Reply:
But we need to keep the value of prev_stat = 2 before the USB is successfully mounted. 
If prev_stat changed before that, Marlin will lose the chance to detect the "first mount" which will make recovery.check() never be called. 

safe_delay() not needed in USB. After DiskIODriver_USBFlash::isInserted() return true, it can be mounted right away.

Reply:
A `prev_stat` of 3 (or changing to a bit mask) will cost less SRAM than a new variable.